doc/SpectatorView.Setup.md

```diff
@@ -53,6 +53,8 @@ Not all spatial alignment strategies support all platforms. See the chart below
 1. Obtain the [MixedReality QR Code Plugin](https://github.com/dorreneb/mixed-reality/releases) and include the associated dll's into the [MixedReality-QRCodePlugin folder](../external/MixedReality-QRCodePlugin).
 2. In the WSA unity player settings, add the **QRCODESTRACKER_BINARY_AVAILABLE** preprocessor directive. (This is located via Build Settings -> Player Settings -> Other Settings -> 'Scripting Defined Symbols')
 
+>Note: Adding QRCODESTRACKER_BINARY_AVAILABLE to the player settings for Android, iOS and HoloLens 2 builds should not generate any issues for other spatial alignment strategies. However, when building for HoloLens 1, the QRCODESTRACKER_BINARY_AVAILABLE needs to be removed from the WSA Player settings. Compilation checks to determine whether a device supports QR Code detection are currently conducted using this QRCODESTRACKER_BINARY_AVAILABLE flag. HoloLens 1 does not support QR Code detection.

```

Comment:
>>Note: A [](start = 0, length = 8)

Nit: I'd lead with "Note: QRCODESTRACKER_BINARY_AVAILABLE should be removed when building for HoloLens 1." followed by the rest of the details.